wordpress-mobile/WordPress-iOS
# NUX: Removes old login code, A/B test, and ReactiveCocoa

This PR ends our A/B test of the new signup flow (and magic links).
The old `LoginViewController` and `CreateAccountAndBlogViewController` is removed, along with the `ReactiveCocoa` pod, and other now unused classes that were a part of old NUX.  The remaining classes are used by the `CreateNewBlogViewController`.

To test:
From a fresh install, sign into wpcom. 
Sign out of wpcom and sign into a self-hosted blog. 
While signed into a self-hosted blog, go to the Me tab and sign into wpcom. 
Go to the My Sites controller. Tap the plus button and create a new blog. 
Confirm that everything works and there are no crashes. 

Needs review: @koke, would you please? 

cc @kwonye
